src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Remove Https configuration
+REM Note that we always return successful code in clean up routine because clean up errors are not fatal
+
+netsh http delete sslcert ipport=0.0.0.0:8084
```

Comment:
It is time to start making ports configurable.  This is now the 2nd cmd file with hardcoded ports.  This comment is not blocking for the initial merge, but please open an issue to revisit this.